mlflow/tracing/fluent.py

```diff
@@ -549,3 +550,133 @@ def f():
         return TRACE_BUFFER.get(last_active_request_id)
     else:
         return None
+
+
+@experimental
+def use_remote_trace(trace: Trace):
```

Comment:
note: Not sure if this naming is clear enough.... more than happy to hear other ideas!

Reply:
`merge_external_trace` might be a bit more descriptive

Reply:
Yea... the reason I didn't use `merge` is it feel it should have a target like `merge_external_trace(from, to)`

Reply:
lemme also open a thread to gather ideas

Reply:
well probably I should add `target` parameter that defaults to current active span.

Reply:
I actually really like this idea. Like REALLY like this idea